tests/network/user_defined_network/test_user_defined_network.py

```diff
@@ -128,23 +126,25 @@ def test_ip_address_is_preserved_after_live_migration(self, vma_udn):
 
     @pytest.mark.polarion("CNV-11434")
     @pytest.mark.single_nic
-    def test_vm_egress_connectivity(self, vmb_udn_non_migratable):
-        assert lookup_iface_status(
-            vm=vmb_udn_non_migratable, iface_name=lookup_primary_network(vm=vmb_udn_non_migratable).name
-        )[IP_ADDRESS]
-        vmb_udn_non_migratable.console(commands=[f"ping -c 3 {PUBLIC_DNS_SERVER_IP}"], timeout=TIMEOUT_1MINUTE)
+    def test_vm_egress_connectivity(self, vmb_udn):
+        assert lookup_iface_status(vm=vmb_udn, iface_name=lookup_primary_network(vm=vmb_udn).name)[IP_ADDRESS]
+        vmb_udn.console(commands=[f"ping -c 3 {PUBLIC_DNS_SERVER_IP}"], timeout=TIMEOUT_1MINUTE)
 
     @pytest.mark.polarion("CNV-11418")
     @pytest.mark.single_nic
-    def test_basic_connectivity_between_udn_vms(self, vma_udn, vmb_udn_non_migratable):
-        target_vm_ip = lookup_iface_status(
-            vm=vmb_udn_non_migratable, iface_name=lookup_primary_network(vm=vmb_udn_non_migratable).name
-        )[IP_ADDRESS]
+    def test_basic_connectivity_between_udn_vms(self, vma_udn, vmb_udn):
+        target_vm_ip = lookup_iface_status(vm=vmb_udn, iface_name=lookup_primary_network(vm=vmb_udn).name)[IP_ADDRESS]
         vma_udn.console(commands=[f"ping -c 3 {target_vm_ip}"], timeout=TIMEOUT_1MIN)
 
     @pytest.mark.polarion("CNV-11427")
     @pytest.mark.single_nic
     @pytest.mark.gating
-    def test_connectivity_is_preserved_after_live_migration(self, vma_udn, server, client):
-        migrate_vm_and_verify(vm=vma_udn)
+    def test_connectivity_is_preserved_during_client_live_migration(self, server, client):
+        migrate_vm_and_verify(vm=client.vm)
```

Comment:
`migrate_vm_and_verify` arg `wait_for_migration_success` default is `True` so this checks connectivity after migration

Reply:
@rnetser , I think the phrasing is correct although the interpretation may be taken into different directions.
The intention is for the connectivity to be preserved during, including the successful end, of the migration.

If there is a problem during the migration, e.g. the connection is dropped, the assertion below will detect it.

Reply:
this is based on the current implementation and makes assump[tioans; leaving this open so we can discuss this offline